scripts/build-docker-image.sh

```diff
@@ -54,10 +35,10 @@ docker tag $ECR_REPO_ENV:$IMAGE_TAG $AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.
 
 
 ##### pushing docker image to ECR ################
-echo Build completed on `date`
-echo Pushing the Docker image...
-docker push $AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$ECR_REPO_ENV:$IMAGE_TAG
-
-####### update ECS service with new image ################
-
-aws ecs update-service --cluster $DEPLOYMENT_ENV-microservice-cluster  --service ab2d-event-service --force-new-deployment
+#echo Build completed on `date`
+#echo Pushing the Docker image...
+#docker push $ACCOUNT_NUMBER.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$ECR_REPO_ENV:$IMAGE_TAG
+#
+######## update ECS service with new image ################
+#
+#aws ecs update-service --cluster $DEPLOYMENT_ENV-microservice-cluster  --service ab2d-event-service --force-new-deployment
```

Comment:
Looks like changes in the right direction here, but I think a cleaner way forward would be to leave the build-docker-image.sh script as it is and create new scripts for this workflow:

- docker-build-push.sh for building and pushing the docker image to ECR with github
- ecs-new-image.sh for pointing the ECS cluster at a new docker image